gallery/src/components/NxNexusPageHeader/NxNexusPageHeaderPage.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import NxNexusPageHeaderExample from './NxNexusPageHeaderExample';
+import NxNexusPageHeaderCustomLogoExample from './NxNexusPageHeaderCustomLogoExample';
+
+const nxNexusPageHeaderExampleCode = require('!!raw-loader!./NxNexusPageHeaderExample').default;
+const nxNexusPageHeaderCustomLogoExampleCode = require('!!raw-loader!./NxNexusPageHeaderCustomLogoExample').default;
+
+const NxNexusPageHeaderPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxNexusPageHeader</code> is a React component encapsulating the Sonatype Nexus
+        branded page header logo and structure. This header should only be used for products in the Sonatype Nexus
+        family.
+      </p>
+      <h3 className="nx-h3 nx-tile__subsection-header">Props</h3>
+      <table className="nx-table nx-table--gallery-props">
+        <thead>
+          <tr className="nx-table-row">
+            <th className="nx-cell nx-cell--header">Prop</th>
+            <th className="nx-cell nx-cell--header">Type</th>
+            <th className="nx-cell nx-cell--header">Required</th>
+            <th className="nx-cell nx-cell--header">Details</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr className="nx-table-row">
+            <td className="nx-cell">productInfo</td>
+            <td className="nx-cell">object</td>
+            <td className="nx-cell">Yes</td>
+            <td className="nx-cell">
+              An object containing at least one field: <code className="nx-code">name</code>, the product
+              name to display in the header. Additionally, this object may contain
+              a <code className="nx-code">meta</code> field which might contain information about the license
+              (as it does in IQ), or other information about the product (e.g. beta, pre-release)
+              and a <code className="nx-code">version</code> field holding the product's version number to display.
```

Comment:
Given that `meta` and `version` are both optional, I think we need three more examples: one with only the `meta`, one with only the `version`, and one with neither.  A non-zero amount of effort went into making sure the spacing and the pipe are correct in each of those cases, so they should be visible proven out (and visually tested).

Reply:
> Given that `meta` and `version` are both optional, I think we need three more examples: one with only the `meta`, one with only the `version`, and one with neither. A non-zero amount of effort went into making sure the spacing and the pipe are correct in each of those cases, so they should be visible proven out (and visually tested).

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/f364b7bfd87c6fc7798d7d00993d5de79c714ff0